exec/java-exec/src/main/resources/rest/storage/list.ftl

```diff
@@ -190,7 +190,7 @@
               <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
               <button type="submit" class="btn btn-primary" onclick="doCreate()">Create</button>
             </div>
-            <input type="hidden" name="csrfToken" value="${model.getCsrfToken()}">
+            <input type="hidden" name="csrfToken" value="${model[0].getCsrfToken()}">
```

Comment:
Not sure why now we get first model, can be there more than one or none?

Reply:
Now we have one model per plugin, but they all contain the same CSRF token. In case when there is no plugins one empty model with the token is created.